runtime/kusama/src/lib.rs

```diff
@@ -1246,33 +1247,57 @@ impl auctions::Config for Runtime {
 	type WeightInfo = weights::runtime_common_auctions::WeightInfo<Runtime>;
 }
 
+type NisCounterpartInstance = pallet_balances::Instance2;
+impl pallet_balances::Config<NisCounterpartInstance> for Runtime {
+	type Balance = Balance;
+	type DustRemoval = ();
+	type RuntimeEvent = RuntimeEvent;
+	type ExistentialDeposit = ConstU128<10_000_000_000>; // One KTC cent
```

Comment:
`KTC` is the name of the counter-part token? And there are going to be 21M of them?